beta/src/pages/apis/createRoot.md

```diff
@@ -25,58 +25,99 @@ const root = createRoot(domNode, options?)
 
 ## Usage {/*usage*/}
 
-Call `createRoot` to create a React root for a <CodeStep step={1}>browser DOM node</CodeStep>.
+### Rendering an app fully built with React {/*rendering-an-app-fully-built-with-react*/}
 
-Then call `render` to display a <CodeStep step={2}>React component</CodeStep>.
+If your app is fully built with React, create a single root for your entire app.
 
-```js [[1, 3, "document.getElementById('root')"], [2, 4, "<App />"]]
-import {createRoot} from 'react-dom/client';
+```js [[1, 4, "document.getElementById('root')"], [2, 5, "<App />"]]
+import { createRoot } from 'react-dom/client';
+import App from './App.js';
 
 const root = createRoot(document.getElementById('root'));
 root.render(<App />);
 ````
 
-Using `createRoot()` to hydrate a server-rendered container is not supported. Use [`hydrateRoot()`](/apis/hydrateRoot) instead.
-
-<Note>
-
-In apps fully built with React, **you will usually only create one "root", once at startup for your entire app**. If you use a framework, it might do this call for you.
-
-</Note>
+You only need to run this code once at startup. It will:
 
-### Rendering a root component {/*rendering-a-root-component*/}
-A React "root" is an object that wraps a DOM node and allows you to display a "root component" inside it.
+1. Find the <CodeStep step={1}>browser DOM node</CodeStep> defined in your HTML.
+2. Display the <CodeStep step={2}>React component</CodeStep> for your app inside.
 
-Once a root is created, you can use it to display a root component using [`root.render`](#root-render).
+Depending on your setup, this code may be in an entry point like `src/index.js`, or even inside a framework.
 
 <Sandpack>
 
+```html index.html
+<!DOCTYPE html>
+<html>
+  <head><title>My app</title></head>
+  <body>
+    <!-- This is the DOM node -->
+    <div id="root"></div>
+  </body>
+</html>
+```
+
 ```js index.js active
-import './styles.css';
-import {createRoot} from 'react-dom/client';
+import { createRoot } from 'react-dom/client';
 import App from './App.js';
+import './styles.css';
 
-const domNode = document.getElementById('root');
-const root = createRoot(domNode);
+const root = createRoot(document.getElementById('root'));
```

Comment:
One bummer about this is that the line overflows the window because it's so long, which is why I broke it into two lines.

Reply:
looks ok on my screen but if you feel strongly, we could just break the line in sandbox. don't want to have different variables between snippet and sandbox